src/System.ServiceModel.Syndication/src/System/ServiceModel/Syndication/Atom10FeedFormatter.cs

```diff
@@ -1027,6 +1024,7 @@ private void ReadLink(XmlReader reader, SyndicationLink link, Uri baseUri)
                     else if (reader.LocalName == Atom10Constants.HrefTag && reader.NamespaceURI == string.Empty)
                     {
                         val = reader.Value;
+                        ns = reader.NamespaceURI;
                     }
```

Comment:
You only set ns on this line and reader.NamespaceURI is guaranteed to be string.Empty. So why record it?

Reply:
Good catch. This is from the previous implementation. I'll remove this and use string.empty directly.